Sources/Widgets/Widgets3D/ImplicitPlaneWidget/index.js

```diff
@@ -43,18 +43,23 @@ function widgetBehavior(publicAPI, model) {
     ) {
       return macro.VOID;
     }
-    isDragging = true;
+
     model.lineManipulator.setWidgetOrigin(model.widgetState.getOrigin());
     model.planeManipulator.setWidgetOrigin(model.widgetState.getOrigin());
     model.trackballManipulator.reset(callData); // setup trackball delta
-    model._interactor.requestAnimation(publicAPI);
-    publicAPI.invokeStartInteractionEvent();
 
+    if (model.dragable) {
+      isDragging = true;
+      model._apiSpecificRenderWindow.setCursor('grabbing');
+      model._interactor.requestAnimation(publicAPI);
+    }
+
+    publicAPI.invokeStartInteractionEvent();
     return macro.EVENT_ABORT;
   };
 
   publicAPI.handleMouseMove = (callData) => {
-    if (isDragging && model.pickable) {
+    if (isDragging && model.pickable && model.dragable) {
```

Comment:
isn't `if (isDragging)` sufficient ?

Reply:
Yes indeed, I've removed `model.pickable && model.dragable` to make code cleaner